NEWS.md

```
@@ -15,12 +15,13 @@
   * Error messages adjustment.
   * Function promoted to official.
 
-**Changes on proposed functions**
-
 * pgr_findCloseEdges
 
   * Error messages adjustment.
   * ``partial`` option is removed.
+  * Function promoted to official.
+
+**Changes on proposed functions**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)